### PR TITLE
FIX Update namespace for LoggerInterface in config, move config to statics, fix delete all button

### PR DESCRIPTION
**Pull request by robbieaverill**

### CODE REVIEW

File: _config/config.yml [hunk 1]
```diff
@@ -2,20 +2,6 @@
...
-SilverLeague\LogViewer\Model\LogEntry:
-  # Whether the cron functionality should run. This does not affect use as a BuildTask.
-  # Note: you need to configure silverstripe/crontask yourself.
-  cron_enabled: true
-  # How often the cron should run (default: 4am daily)
-  cron_schedule: 0 4 * * *
-  # The maximum age in days for a LogEntry before it will be removed
-  max_log_age: 30
-  # Which Monolog\Logger levels (numeric) to start handling from (see class for examples)
-  minimum_log_level: 300 # WARNING
```
Comment (Firesphere) [inline _config/config.yml:21]: Can you move this to an example or docs, as a reference for users

File: src/Forms/GridField/GridFieldClearAllButton.php [hunk 2]
```diff
@@ -39,13 +39,13 @@ public function __construct($targetFragment = 'after')
...
     public function getHTMLFragments($gridField)
     {
-        $button = GridField_FormAction::create($gridField, 'clear', 'Clear all', 'clear', null);
-        $button->setAttribute('data-icon', 'clear-all-logs');
-        $button->addExtraClass('font-icon-trash-bin action_clear action action-delete btn btn-secondary');
-        $button->setForm($gridField->getForm());
+        $button = GridField_FormAction::create($gridField, 'clear', 'Clear all', 'clear', null)
+            ->setAttribute('data-icon', 'clear-all-logs')
+            ->addExtraClass('font-icon-trash-bin action_clear no-ajax action-delete btn btn-secondary')
+            ->setForm($gridField->getForm());
```
Comment (Firesphere) [inline src/Forms/GridField/GridFieldClearAllButton.php:45]: 👍 
I love chaining :D

File: src/Model/LogEntry.php [hunk 3]
```diff
@@ -14,29 +14,53 @@
...
+     */
+    private static $max_log_age = 30;
+
+    /**
+     * Which Monolog\Logger levels (numeric) to start handling from (see class for examples)
+     *
+     * @config
+     * @var integer
+     */
+    private static $minimum_log_level = 300;
```
Comment (Firesphere) [inline src/Model/LogEntry.php:61]: Personal opinion: I like single-line statics above the `db` declarations etc. Not really relevant, I'm okay with leaving it as well

**Review by Firesphere — CHANGES_REQUESTED** [review]
Some minor things

File: _config/config.yml (hunk 1 above)
Comment (robbieaverill) [reply _config/config.yml:21]: Yeah, it's in the Configuration section of the readme

**Review by Firesphere — APPROVED** [review]

**Pull request merged**